### PR TITLE
feature/performance

### PR DESCRIPTION
**Pull request by oozoofrog**
@gmlwhdtjd we can speed up here together

- add performance check script
 
> ./run-performance.sh
...
34.1.ms blah blah
...

### CODE REVIEW

**Comment by gmlwhdtjd** [comment]
Now, this pr seems to be working on the build speed and runtime speed of our package. Of course, that is also a very important part. But what I'm really interested in is the part about whether the build speed of the user project is not lowered due to type estimation when using our package.

**Comment by oozoofrog** [comment]
> Now, this pr seems to be working on the build speed and runtime speed of our package. Of course, that is also a very important part. But what I'm really interested in is the part about whether the build speed of the user project is not lowered due to type estimation when using our package.

I understand. anyway, some parts of codes being very dramatically speed up.

**Review by gmlwhdtjd — APPROVED** [review]

File: Sources/SwiftLayout/Core/Activator/Activation.swift [hunk 1]
```diff
@@ -10,14 +10,14 @@ public final class Activation: Hashable {
     
     typealias Constraints = Set<WeakReference<NSLayoutConstraint>>
     
-    var viewInfos: ViewInformationSet
+    var viewInfos: [ViewInformation]
```
Comment (oozoofrog) [inline Sources/SwiftLayout/Core/Activator/Activation.swift:13]: 👍

**Review by gmlwhdtjd — APPROVED** [review]
👍

**Pull request merged**